init.go

```diff
@@ -156,7 +156,11 @@ func postInitPlugins(event apid.Event) {
 
 		tokenManager = createTokenManager()
 
-		go bootstrap()
```

Comment:
bootstrap function is still in apigee_sync. The diff should be there.  All of the changes are in code review.